models/userDao.js

```diff
@@ -100,7 +103,22 @@ const getUserProfile = async (userId) => {
   );
 };
 
+
+const getUserByInfo = async (userId) => {
```

Comment:
```suggestion
const getUserById = async (userId) => {
```
info로 user를 get하는 것이 아닌, Id로 user를 get하고 있기 때문에 수정이 필요합니다.

Reply:
넵 수정했습니다 !